### DISCUSSION TITLE
Does JuiceFS not rely on object storage or hdfs and instead become a separate object storage to replace these products?

### DISCUSSION THREAD

**Discussion by smileyboy2019** [juicedata/juicefs#4091] [General]
Because there is no hdfs or object storage, can we use local storage to make hdfs, support object services, compare with mini, support file size monitoring, traffic monitoring, dynamic expansion, data 3 backup, and the concept of buckets and volumes. Can images and videos be uploaded for accessing image and video websites?

**Comment by davies** [comment]
We don't have a plan to build a object store for JuiceFS right now.
